packages/fx-core/src/plugins/resource/sql/plugin.ts

```diff
@@ -289,7 +292,10 @@ export class SqlPluginImpl {
 
     await managementClient.deleteLocalFirewallRule();
 
-    TelemetryUtils.sendEvent(Telemetry.stage.postProvision, true);
+    TelemetryUtils.sendEvent(Telemetry.stage.postProvision,
+      true,
+      { [Telemetry.properties.skipAddingUser]: this.config.skipAddingUser ? Telemetry.resultYes : Telemetry.resultNo }
```

Comment:
Maybe create new constants for the skipAddingUser property? Though the string we're going to set is same, but `Telemetry.resultYes` represents different meaning.

Reply:
I thought yes and no are the general keywords for telemetry to search and filter. Maybe I can change the name for the constants to make the meaning clear.